### PR TITLE
feat(radio): refactor check component to remove radio and create radio component

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/1206

### CODE REVIEW

File: src/patternfly/components/Check/examples/check-example.hbs [hunk 1]
```diff
@@ -1,20 +1,20 @@
 {{#> check}}
-  {{#> check-input check-input--attribute='type="checkbox" id="check1" name="examplecheck1"'}}{{/check-input}}
+  {{#> check-input check-input--attribute='id="check1" name="exampleChecks"'}}{{/check-input}}
```
Comment (matthewcarleton) [inline src/patternfly/components/Check/examples/check-example.hbs]: should we keep the unique names for the checkboxes? I don't you commonly want checkboxes to share names do you?

File: src/patternfly/components/Radio/examples/radio-example.hbs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+{{#> radio}}
+  {{#> radio-label radio-label--attribute='for="radio1-rev"'}}Radio 1 Reversed{{/radio-label}}
+  {{#> radio-input radio-input--attribute='id="radio1-rev" name="exampleRadios"'}}{{/radio-input}}
+{{/radio}}
+{{#> radio radio--type="label" radio--attribute='for="radio2"'}}
+  {{#> radio-input radio-input--attribute='id="radio2" name="exampleRadios"'}}{{/radio-input}}
+  {{#> radio-label radio-label--type="span"}}Radio 2 label wraps input{{/radio-label}}
+{{/radio}}
+{{#> radio}}
+  {{#> radio-input radio-input--attribute='id="radio3" name="exampleRadios" radioed'}}{{/radio-input}}
```
Comment (matthewcarleton) [inline src/patternfly/components/Radio/examples/radio-example.hbs]:
```suggestion
  {{#> radio-input radio-input--attribute='id="radio3" name="exampleRadios" checked'}}{{/radio-input}}
```

File: src/patternfly/components/Radio/examples/radio-example.hbs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  {{#> radio-label radio-label--attribute='for="radio3"'}}Radio 3 radioed{{/radio-label}}
```
Comment (matthewcarleton) [inline src/patternfly/components/Radio/examples/radio-example.hbs]:
```suggestion
  {{#> radio-label radio-label--attribute='for="radio3"'}}Radio 3 checked{{/radio-label}}
```

**Comment by mcoker** [comment]
@matthewcarleton thanks! updated.

**Pull request merged**